tests/system/action/motion_category/test_number_motions.py

```diff
@@ -0,0 +1,539 @@
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        self.check_helper("motion/1", "3")
+        self.check_helper("motion/2", "2")
+        self.check_helper("motion/3", "1")
+
+    def test_stop_prefix_lookup_at_main_category(self) -> None:
+        self.create_model(
+            "meeting/1",
+            {
+                "name": "name_meeting_1",
+                "motion_category_ids": [1, 2],
+                "motion_ids": [1],
+            },
+        )
+        self.create_model(
+            "motion_category/1",
+            {
+                "name": "category_1",
+                "meeting_id": 1,
+                "parent_id": None,
+                "child_ids": [2],
+                "motion_ids": [],
+                "prefix": "A",
+            },
+        )
+        self.create_model(
+            "motion_category/2",
+            {
+                "name": "category_2",
+                "meeting_id": 1,
+                "parent_id": 1,
+                "child_ids": [],
+                "motion_ids": [1],
+            },
+        )
+        self.create_model(
+            "motion/1",
+            {"title": "m1", "category_id": 2, "meeting_id": 1, "category_weight": 100},
+        )
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.number_motions",
+                    "data": [
+                        {
+                            "id": 2,
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        self.check_helper("motion/1", "1")
+
+    def test_invalid_id(self) -> None:
+        self.create_model(
+            "meeting/1",
+            {
+                "name": "name_meeting_1",
+                "motion_category_ids": [1, 2],
+                "motion_ids": [1],
+            },
+        )
+        self.create_model(
+            "motion_category/1",
+            {
+                "name": "category_1",
+                "meeting_id": 1,
+                "parent_id": None,
+                "child_ids": [2],
+                "motion_ids": [],
+                "prefix": "A",
+            },
+        )
+        self.create_model(
+            "motion_category/2",
+            {
+                "name": "category_2",
+                "meeting_id": 1,
+                "parent_id": 1,
+                "child_ids": [],
+                "motion_ids": [1],
+            },
+        )
+        self.create_model(
+            "motion/1",
+            {"title": "m1", "category_id": 2, "meeting_id": 1, "category_weight": 100},
+        )
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.number_motions",
+                    "data": [
+                        {
+                            "id": "222",
```

Comment:
This is a string - ok, catched by the validation. But add another test with the same setup, but `"id": 222` and see, that this also returns a valid error (Like "Motion 222 does not exists").

Reply:
222 would be a motion_category. Changed it into an int.